net/upnp/src/etc/inc/plugins.inc.d/miniupnpd.inc

```diff
@@ -41,6 +41,8 @@ function miniupnpd_firewall($fw)
 
     $fw->registerAnchor('miniupnpd', 'rdr');
     $fw->registerAnchor('miniupnpd', 'fw');
+    $fw->registerAnchor('miniupnpd', 'nat', '', 'head');
```

Comment:
Totally agree - I guess the other situation is that (automatic/manual) NAT rules could potentially break UPNP without any user notification if it wasn't at the top.

Perhaps some kind of warning on the opnsense UPNP plugin page? I imagine that would be sufficient since a user needs to go and enable the UPNP service manually after install anyway.

Reply:
let's merge this one and work on the other one to include some sort of sensible warning